### ISSUE TITLE
graph api is hard coded to commercial cloud

### ISSUE THREAD

**Issue by truthersby-bct** [seanfisher/passport-microsoft#42]
In order to use the gov cloud (microsoft.us domains) I can almost get everything working by overriding the optional urls:
```
 // [Optional] The authorization URL. Defaults to `https://login.microsoftonline.com/${tenant}/oauth2/v2.0/authorize`
authorizationURL: 'https://login.microsoftonline.us/${tenant}/oauth2/v2.0/authorize',
// [Optional] The token URL. Defaults to `https://login.microsoftonline.com/${tenant}/oauth2/v2.0/token`
tokenURL: https://login.microsoftonline.us/${tenant}/oauth2/v2.0/token
```

However, the graph API is hard coded and I can't change it to the .us version
https<k>://graph.microsoft.**us**/${strategy._graphApiVersion}/me/

I was able to edit the file it to .us for testing and login works, that was the only difference to get GOV cloud regions working.

Can that URL be exposed as a configuration option as well?
Thank you!

**Comment by seanfisher** [comment]
I believe this should be resolved once this PR is complete: https://github.com/seanfisher/passport-microsoft/pull/41

**Comment by truthersby-bct** [comment]
Awesome, that would resolve this for me. closing

**Comment by seanfisher** [comment]
@truthersby-bct This was released in [2.1.0](https://github.com/seanfisher/passport-microsoft/releases/tag/v2.1.0)
